src/Symfony/Component/HttpFoundation/Tests/ResponseHeaderBagTest.php

```diff
@@ -57,7 +57,7 @@ public function testCacheControlHeader()
         $this->assertFalse($bag->hasCacheControlDirective('max-age'));
 
         $bag = new ResponseHeaderBag(['Expires' => 'Wed, 16 Feb 2011 14:17:43 GMT']);
-        $this->assertEquals('private, must-revalidate', $bag->get('Cache-Control'));
+        $this->assertEquals('must-revalidate, private', $bag->get('Cache-Control'));
```

Comment:
i am actually not sure why the order changes, but the order has no semantical meaning